src/coreclr/tools/aot/crossgen2/crossgen2.props

```diff
@@ -41,12 +41,19 @@
     <Compile Include="..\..\Common\InstructionSetHelpers.cs" Link="InstructionSetHelpers.cs" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <_LibPrefix Condition="'$(CrossHostArch)' == 'true'">$(LibPrefix)</_LibPrefix>
+    <_LibPrefix Condition="'$(CrossHostArch)' != 'true'">$(HostLibPrefix)</_LibPrefix>
+    <_LibSuffix Condition="'$(CrossHostArch)' == 'true'">$(LibSuffix)</_LibSuffix>
+    <_LibSuffix Condition="'$(CrossHostArch)' != 'true'">$(HostLibSuffix)</_LibSuffix>
+  </PropertyGroup>
```

Comment:
To confirm: we need this because we're building a crossgen2 to run on Windows that targets Android?

Reply:
Correct, or mac